README.md

```diff
@@ -3,29 +3,79 @@
 
 This plugin allows you to test **Electron** applications with [TestCafe](http://devexpress.github.io/testcafe).
 
-## Install
+## Getting Started
+
+### Installation
 
 ```
 npm install testcafe-browser-provider-electron
 ```
 
-## Usage
+### Testing a JavaScript Application
 
-First, create a `.testcafe-electron-rc` file in the root directory of your Electron app. For more info, see the [Configuration](#configuration) section.
+If you have a JavaScript application that runs in Electron, you need to go through the following steps to set up testing.
 
-```
-{
-  "mainWindowUrl": "./index.html"
-}
-```
+1. Create a `.testcafe-electron-rc` file in the root application directory. Include the following settings to this file.
 
-When you run tests from the command line, specify the path to the application root directory prefixed with "electron:" :
+    ```json
+    {
+      "mainWindowUrl": "./index.html"
+    }
+    ```
 
-```
-testcafe "electron:/home/user/electron-app" "path/to/test/file.js"
-```
+    The `mainWindowUrl` option specifies the path to a file that was loaded as the application startup page. TestCafe waits for Electron to load this page and then runs tests. If a relative path is specified, it is resolved from the `.testcafe-electron-rc` file location.
+
+    For information about other options, see the [Configuration](#configuration) section.
+
+2. Install the Electron module of the required version.
+
+    ```sh
+    npm install electron@latest
+    ```
+
+    The command above installs the latest version of the Electron executable.
+
+3. Run tests defining the path to the configuration file with a browser provider postfix `electron:<path_to_testcafe-electron-rc_directory>`.
```

Comment:
Define the path to the config file. Use browser provider postfix:
`electron:<path_to_testcafe-electron-rc_directory>`.
Then run tests.